R/init-package.R

```diff
@@ -18,6 +18,4 @@ initPackage <- function() {
   apiConfig$pkParametersFilePath <- filePathFor("OSPSuite.PKParameters.xml")
 
   rClr::clrCallStatic("OSPSuite.R.Api", "InitializeOnce", apiConfig$ref)
-  dimensionTask <- getNetTask("DimensionTask")

```

Comment:
Why do you remove "DimensionTask"? I was counting on it for the `utilities-dimensions`. 

Btw. I only was creating it the first time it was requested:

In ospsuite-env (esqlabsEnv in my code):

```
# NetTask "DimensionTask" cached for performance benefits. Created the first time it is requested.
esqlabsEnv$DimensionTask <- NULL

```

Then a function somewhere:

```
#' Return an instance of the .NET Task "DimensionTask".
#'
#' @return An instance of the Task
getDimensionTask <- function() {
  dimTask <- esqlabsEnv$DimensionTask
  if (is.null(dimTask)) {
    dimTask <- getNetTask("DimensionTask")
    esqlabsEnv$DimensionTask <- dimTask
  }
  return(dimTask)
}
```

Reply:
I don't remove it. Instatntiating object in the init creates weird error messages during build.

Can you quantify ` cached for performance benefits.`.

Reply:
I think you are looking at sthg else. :)

Reply:
but I am going to add it to core because it's probably a good optimization since this is used quite often